### ISSUE TITLE
Проблема с kapt3

### ISSUE THREAD

**Issue by morder** [Arello-Mobile/Moxy#144]
В проекте используются модули 
В модуле указываем в gradle файле
kapt {
    arguments {
        arg("moxyReflectorPackage", "com.some.package")
    }
}

При переходе на новую версию Kotlin 1.1.3-2
Нужно указать вручную apply plugin: 'kotlin-kapt', это применение kapt3, после которого проект перестает собираться 
Не генерируется класс com.some.package.MoxyReflector.

**Comment by xanderblinov** [comment]
Добрый день, @morder извините, что долго не отвечали.

Попробуйте добавить 
`kapt {
    generateStubs = true
}`

Также для нового kapt необходимо подключать 

`apply plugin: 'kotlin-kapt'`

**Comment by morder** [comment]
Все что вы написали я пробовал, не помогает.
Я плотно исследовал эту проблему, в итоге пришел к тому что в нашем модуле в котором тоже используется мокси и котлин, не вызывается кодогенерация а соответственно не создается класс MoxyReflector который подключается в app. На данный момент пришлось отказаться от мокси в модуле.

**Comment by xanderblinov** [comment]
Вы смотрели как работать с несколькими модулями, содержащими Moxy?

В настоящий момент этот стек используется в одном из рабочих проектов (несколько мокси модулей и kotlin)

**Comment by morder** [comment]
Да, конечно. Со старым kapt проблем не было, проблема только с новым kapt3
